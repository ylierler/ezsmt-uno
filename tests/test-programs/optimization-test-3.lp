a.
{b}.
:~a.[1]
:~a, not b. [11]
