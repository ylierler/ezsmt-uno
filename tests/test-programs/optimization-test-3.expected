a 
 12

b a 
 1